### PR TITLE
rename binary

### PR DESCRIPTION
**Pull request by z7658329**

### CODE REVIEW

**Comment by z7658329** [comment]
image

image

**Review by javeme — APPROVED** [review]
LGTM

**Review by imbajin — APPROVED** [review]

File: hugegraph-dist/pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
     <properties>
         <release.name>${project.parent.artifactId}</release.name>
-        <final.name>${release.name}-${project.version}</final.name>
+        <final.name>apache-${release.name}-incubating-${project.version}</final.name>
```
Comment (imbajin) [inline hugegraph-dist/pom.xml:17]: @z7658329 should use one param for these modules?

seems we just define it in root-pom is OK?

File: pom.xml [hunk 2]
```diff
@@ -61,7 +61,7 @@
     <properties>
         <revision>1.0.0</revision>
         <release.name>${project.artifactId}</release.name>
-        <final.name>${release.name}-${project.version}</final.name>
+        <final.name>apache-${release.name}-incubating-${project.version}</final.name>
```
Comment (imbajin) [inline pom.xml:64]: the child module will extend it automatically

**Pull request merged**